universe/ignore_records.go

```diff
@@ -92,6 +98,88 @@ func (i *IgnoreTuple) Record() tlv.Record {
 	)
 }
 
+// Encode serializes the IgnoreTuple to the given io.Writer.
+func (i *IgnoreTuple) Encode(w io.Writer) error {
+	stream, err := tlv.NewStream(i.Record())
+	if err != nil {
+		return err
+	}
+
+	return stream.Encode(w)
+}
+
+// Bytes returns the serialized IgnoreTuple.
+func (i *IgnoreTuple) Bytes() ([]byte, error) {
+	var buf bytes.Buffer
+	err := i.Encode(&buf)
+	if err != nil {
+		return nil, err
+	}
+
+	return buf.Bytes(), nil
+}
+
+// Digest returns the SHA256 digest of the TLV serialized IgnoreTuple.
+func (i *IgnoreTuple) Digest() ([sha256.Size]byte, error) {
+	var zero [sha256.Size]byte
+
+	ignoreBytes, err := i.Bytes()
+	if err != nil {
+		return zero, err
+	}
+
+	digest := sha256.Sum256(ignoreBytes)
+	return digest, nil
+}
+
+// GenSignedIgnore generates a Schnorr signature over the IgnoreTuple using the
+// provided SignerClient and key locator.
+func (i *IgnoreTuple) GenSignedIgnore(ctx context.Context,
```

Comment:
Some basic unit tests around this would be nice. Can take code from here, which is very similar: https://github.com/lightninglabs/taproot-assets/blob/a132dea7b64ca5ec2c1cd4bad27a9acc7e885566/proof/uni_commitments_test.go#L119

And perhaps add a `Verify` method as well, since that will certainly be used at some point?